File: Question5.c
// Write a program that takes an integer as input and returns an integer with reversed digit ordering
#include <stdio.h>

int reverseInt(int num) {
    int reversed = 0;
    if (num < 0) {
        num = -num; 
    }
    while (num > 0) {
        reversed = reversed * 10 + (num % 10);
        num /= 10;
    }
    return reversed; 
}

int main() {
    int num;
    printf("Enter an integer: ");
    scanf("%d", &num);
    printf("Reversed: %d\n", reverseInt(num));
    
    return 0;
}

File: Question.5.py
'''
Write a program that takes an integer as input and returns an integer with reversed digit 
ordering. 
'''
import numpy as np
def reverse_int(n):
    arr = np.array([int(num) for num in str(n)])  
    arr = arr[::-1]  
    reversed_num = int("".join(map(str, arr))) 
    return reversed_num

num = 456373376
result = reverse_int(num)
print(result)